assets/groups.yaml

```diff
@@ -0,0 +1,18 @@
+#acoustic_layer:
+#  - acoustic_oos
+#lexical_layer:
+#  - lexical_oos
+oos_intents:
```

Comment:
`groups.yaml` captures all the intent groups that need to be batched and evaluated together. each new group is associated with a list of existing intents. this is different from aliasing in that the intents in a group are still treated individually, and their numbers are averaged _after_ evaluation